src/styles/style.css

```diff
@@ -14,7 +14,7 @@
   --gatsbyMaxWidth-3xl: 48rem;
   --gatsbyMaxWidth-4xl: 56rem;
   --gatsbyMaxWidth-full: "100%";
-  --gatsbyMaxWidth-wrapper: var(--gatsbyMaxWidth-2xl);
+  --gatsbyMaxWidth-wrapper: 680px;
```

Comment:
This will cause problems with mobile devices, see how `--gatsbyMaxWidth-2xl` is calculated

Reply:
updated